### DISCUSSION TITLE
Is there a way to change the hl for the active group icon?

### DISCUSSION THREAD

**Discussion by Ajaymamtora** [lucobellic/edgy-group.nvim#28] [Q&A] [answered]
It would be good if we could indicate which group is active, atm all the icons are the same colour:

<img width="842" alt="Screenshot 2024-09-21 at 16 26 27" src="https://github.com/user-attachments/assets/924c2a31-a51e-49ea-b232-7fa50635f9ea">

**Comment by lucobellic** [comment, marked as answer]
The `active` and `inactive` colors option with `colored` set to true should be sufficient.

Defaults options set the same highlight for both.

```lua
opts = {
  statusline = {
    colored = true,
    colors = {
      active = 'Normal',
      inactive = 'Normal',
    }
  }
}
```

**Reply by Ajaymamtora** [reply]
Thanks that works
